### PR TITLE
Update MCF build script to install packages.

### PR DESCRIPTION
**Pull request by mmb**
@drnic

### CODE REVIEW

**Comment by drnic** [comment]
I had continued on the script at https://github.com/drnic/micro/tree/end-to-end-script but was pseudo-blocked from continuing by the raised bug; and fear of the 50 hours it takes to run the script without btrfs installed yet.

File: scripts/create_mcf.sh [hunk 1]
```diff
@@ -35,14 +35,26 @@ if [[ ! -f ${ISO_NAME} ]]; then
...
+zip \
+zlib1g-dev
+
 export UBUNTU_MIRROR=http://localhost:9999/ubuntu
 
 # Install a gem if $UPGRADE or if gem not yet installed
 function install_gem() {
   gem_name=$1
   if [[ ("${UPGRADE}X" != "X") || "$(gem list $gem_name | grep $gem_name)X" == "X" ]]; then
-    sudo gem install $gem_name --no-ri --no-rdoc
```
Comment (drnic) [inline scripts/create_mcf.sh:45]: hmm, sudo is necessary for gems that have executables being installed into system rubies (which is how I setup the inception VM in bosh-bootstrap).

Is there a way to detect if sudo is necessary for installing gems? Perhaps we add a $SYSTEM_RUBY variable?

Comment (drnic) [inline scripts/create_mcf.sh:45]: I've never quite figured out what parts of the bosh-bootstrap actually need to be run as root and what don't. Perhaps none of it needs to be root; which would be great. Perhaps we can walk through the scripts. I always assumed that building stemcells required you to be root user.

Comment (mmb) [inline scripts/create_mcf.sh:45]: I was using per-user rbenv so sudo made it not work. We should document the recommended way to install ruby on the inception VM (which may be the way bosh-bootstrap does it). Some kind of detection sounds good.

I've always run this script as a regular user. It seems like everything that needs sudo will sudo.

**Comment by mmb** [comment]
Maybe we can merge the two this week.

**Pull request merged**